pkgs/by-name/wi/wideriver/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = "wideriver";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-Jn3DfflDlMSZMlSRBfzQeEUKgYLctd/PiQRWH8uwtlU=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    wayland-scanner
```

Comment:
Should be in `nativeBuildInputs`?

Reply:
You are right, since this tool just generates glue code for compilation.

Reply:
Moved it, thanks!